stopcovid/dialog/engine.py

```diff
@@ -221,6 +222,14 @@ def _validate_registration(
                 return [UserValidationFailed(**base_args)]
         return None
 
+    # let the system test user text "opus" to restart the demo without finishing it
+    def _demo_user_demo_requested(
+        self, dialog_state: DialogState, base_args: Dict[str, Any]
+    ) -> Optional[List[DialogEvent]]:
+        if dialog_state.user_profile.is_demo and self.content_lower == "opus":
+            return [DemoRequested(**base_args)]
+        return None
```

Comment:
I think I'd be fine with just allowing "opus" to trigger a demo at any time even if you're in the middle of another drill. Would that simplify things? 🤷‍♂️

Reply:
It definitely would, but im wondering if there's ever a case where someone would text opus without actually wanting to get the demo?

Reply:
yeah I can't be 100% sure, but I also can't really imagine us asking people to say "opus" in like a covid drill or something, or ESL. Only place is maybe inside the demo drill itself? Either way is good though 👍